features.bzl

```diff
@@ -37,6 +37,9 @@ _flags = struct(
 _rules = struct(
     # Whether TemplateDict#add_joined allows the map_each callback to return a list of strings (#17306)
     template_dict_map_each_can_return_list = ge("6.1.0"),
+
+    # Whether repository_ctx#download has the block parameter, allowing parallel downloads
```

Comment:
Please add a reference to the Bazel PR adding the feature.

Reply:
I can't find a merged PR (there's an unmerged one [here](https://github.com/bazelbuild/bazel/pull/20309). I can add a reference to the issue, though: https://github.com/bazelbuild/bazel/issues/19674

Reply:
This is the commit: https://github.com/bazelbuild/bazel/commit/73c1a1e382cdd99766e4e11bb917e9672c11b5f3